app/controllers/comfy/admin/web_settings_controller.rb

```diff
@@ -7,6 +7,14 @@ def edit
 
   def update
     if Subdomain.current.update(subdomain_params)
+      if Subdomain.current.enable_2fa 
```

Comment:
@ajaystha42 this shouldn't happen in the controller. Move to the subdomain model and call it in an active record hook for when the `enable_2fa_changed?` changes

Reply:
okay, I will move it to subdomain model